contracts/protocol/clients/voucher/BosonVoucher.sol

```diff
@@ -524,30 +624,36 @@ contract BosonVoucher is IBosonVoucher, BeaconClientBase, OwnableUpgradeable, ER
      * a new buyer account will be created and associated with
      * the exchange.
      *
-     * @param from - the address from which the voucher is being transferred
-     * @param to - the address to which the voucher is being transferred
-     * @param tokenId - the tokenId of the voucher that is being transferred
+     * @param _from - the address from which the voucher is being transferred
+     * @param _to - the address to which the voucher is being transferred
+     * @param _tokenId - the tokenId of the voucher that is being transferred
      */
     function _beforeTokenTransfer(
-        address from,
-        address to,
-        uint256 tokenId
+        address _from,
+        address _to,
+        uint256 _tokenId
     ) internal override {
         // Update the buyer associated with the voucher in the protocol
         // Only when transferring, not minting or burning
-        if (from == owner()) {
-            if (premintStatus.committable) {
-                // If this is a transfer of premited token, treat it differently
+        if (_from == owner()) {
+            if (_premintStatus.committable) {
+                // Set the preminted token as committed
+                _committed[_tokenId] = true;
```

Comment:
this line seems new, do we have a test to cover it ?

Reply:
Yes. There is no explicit test for value of `_committed`, since it's private, but there are at least three different tests that works only if `_committed` is set correctly.
```
Second transfer should behave as normal voucher transfer
Returns true owner if token exists - via preminted voucher transfer.
Should skip all vouchers were already commited
```